src/internal/pfsdb/pfsdb.go

```diff
@@ -28,16 +28,28 @@ var ReposTypeIndex = &col.Index{
 	},
 }
 
+func ReposNameKey(repo *pfs.Repo) string {
```

Comment:
It’s called that because it is the key for the index named `ReposNameIndex`; I think that in turn is named that because there will be multiple repos per name (because of .spec & .meta repos).